src/attributes.jl

```diff
@@ -300,6 +300,42 @@ function MOI.get(m::Optimizer, attr::MOI.VariablePrimal,
     return output
 end
 
+#### Variable basis status
+
+function _basis_status(x::Stakey)
+    if x == MSK_SK_BAS
+        return BASIC
+    elseif x == MSK_SK_SUPBAS
+        return SUPER_BASIC
+    elseif x == MSK_SK_LOW
+        return NONBASIC_AT_LOWER
+    elseif x == MSK_SK_UPR
+        return NONBASIC_AT_UPPER
+        # FIXME which one is `NONBASIC` ?
```

Comment:
I don't think Mosek has this. It will either be nonbasic at a bound or super basic.